### PR TITLE
feat: missing cred UI

### PR DESCRIPTION
**Pull request by wadeking98**
# Summary of Changes

Improved UI for missing credentials. Added elements to clearly separate credentials that exist in the wallet and ones that are missing:
image
 

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: packages/legacy/core/App/screens/ProofRequest.tsx [hunk 1]
```diff
@@ -407,25 +404,19 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
...
-                    </Text>
-                  ) : (
-                    <Text style={styles.headerText} testID={testIdWithKey('HeaderText')}>
-                      {t('ProofRequest.YouDoNotHaveDataPredicate')}{' '}
-                      <Text style={[TextTheme.title]}>{proofConnectionLabel || t('ContactDetails.AContact')}</Text>
-                    </Text>
-                  )}
+
+                  <Text style={styles.headerText} testID={testIdWithKey('HeaderText')}>
+                    {t('ProofRequest.YouDoNotHaveDataPredicate')}{' '}
```
Comment (jleach) [inline packages/legacy/core/App/screens/ProofRequest.tsx:417]: Just checking `{' '}` is needed here. Some times my VSCode inserts stuff like this when formatting.

File: packages/legacy/core/App/screens/ProofRequest.tsx [hunk 2]
```diff
@@ -407,25 +404,19 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
...
+                    <Text style={[TextTheme.title]}>{proofConnectionLabel || t('ContactDetails.AContact')}</Text>
```
Comment (jleach) [inline packages/legacy/core/App/screens/ProofRequest.tsx:418]: `??` or `||`?

**Review by jleach — APPROVED** [review]

**Comment by jleach** [comment]
Approved. My two comments are more of a curiosity than anything else.

File: packages/legacy/core/App/screens/ProofRequest.tsx (hunk 1 above)
Comment (wadeking98) [reply packages/legacy/core/App/screens/ProofRequest.tsx:417]: not really sure, I didn't edit those lines I just updated the code around them so they show up as a change

**Pull request merged**